installation/synology.md

```diff
@@ -3,14 +3,133 @@ layout: documentation
 title: Synology DiskStation
 ---
 
-# Synology DiskStation
+# Synology DiskStation DSM 7.0 and higher
+
+The [DiskStation by Synology](https://www.synology.com/en-us/dsm) is a famous NAS server solution for your home, allowing the installation of additional packages. 
+For versions of DSM before 7.0 please refer to the second half of this page.
+
+## Prerequisite - prepare the Synology OS for OpenHAB
+
+Synology NAS is based on Linux but it has limitations since it is not a full server.
+There are some shortcuts you have to take to make it work.  
+For the use of OpenHAB you need some preparation steps.
+For this create a ssh session into your NAS.
+For reference how to do that: I found this [youtube tutorial](https://www.youtube.com/watch?v=BCCIMRbAUp8).
+When you are in the secured shell first thing to do is to create the user openhab.
+```bash
+sudo synouser --add openhab habopen
+sudo synogroup --add openhab openhab
+```
+If you are going to use an USB like a Z-Wave dongle add another group for openhab and change the port access for ttyACM0 if that is the port for your USB device
+```bash
+sudo synogroup --add dialout openhab
+sudo chown root:dialout /dev/ttyACM0
+sudo chmod g+rw /dev/ttyACM0
+```
+Then for installation purposes run the following command:
+```bash
+sudo id openhab
+```
+The result will something like
+```bash
+uid=1032(openhab) gid=100(users) groups=100(users),65537(dialout),65539(openhab)
+```
+In this case we need the 1032 as the user and the 65539 as the group. Write them down.
+
+## Docker
+
+DSM 7 does not longer support java8 directly and therefor OpenHAB has to run in a Docker container. 
+And that also makes the installation easier to maintain. 
+It works out of the box a bit different then the normal Docker installation as described in de OpenHAB documententation.
+Docker is a containerization platform and is used to run lightweight containers. 
+These containers require a very little amount of memory and system resources to run. 
+Synology NAS has official support for Docker.
+To use Docker, you need to install the Docker app from the Synology Web GUI.
+
+- Login and open the DiskStation Manager.
+- Go to Main Menu → Package Center.
+- Search for "Docker" and install.
+- Click "Open".
+- Go to "Registry", here are all the available Docker images and search for openhab.
+- Choose the "openhab/openhab" image and click "Download". Select "latest" since that is the latest stable version of OpenHAB. The Docker image is added to "Image"
```

Comment:
```suggestion
- Choose the "openhab/openhab" image and click "Download". Select "latest" since that is the latest stable version of openHAB. The Docker image is added to "Image"
```